scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -34,35 +42,69 @@ workflow GvsCreateVATfromVDS {
-            output_annotated_file_name = "${input_vcf_name}_annotated",
-            nirvana_data_tar = nirvana_data_directory,
-            custom_annotations_file = StripCustomAnnotationsFromSitesOnlyVCF.output_custom_annotations_file,
+            intervals = interval_list,
+            ref_fasta = reference,
+            ref_fai = reference_index,
+            ref_dict = reference_dict,
+            scatter_count = effective_scatter_count,
+            output_gcs_dir = output_path + "intervals",
+            split_intervals_disk_size_override = split_intervals_disk_size_override,
+            split_intervals_mem_override = split_intervals_mem_override,
     }
 
-    call PrepVtAnnotationJson {
-        input:
-            annotation_json = AnnotateVCF.annotation_json,
-            output_file_suffix = "${input_vcf_name}.json.gz",
-            output_path = output_path,
-    }
+    String sites_only_vcf_basename = basename(basename(input_sites_only_vcf, ".gz"), ".vcf")
+
+    scatter(i in range(length(SplitIntervals.interval_files))) {
+        String interval_file_basename = basename(SplitIntervals.interval_files[i], ".interval_list")
+        String vcf_filename = interval_file_basename + "." + sites_only_vcf_basename
+
+        call Utils.SelectVariants {
+            input:
+                input_vcf = IndexVcf.output_vcf,
+                input_vcf_index = IndexVcf.output_vcf_index,
+                interval_list = SplitIntervals.interval_files[i],
+                output_basename = vcf_filename
+        }
+
+        call StripCustomAnnotationsFromSitesOnlyVCF {
+            input:
+                input_vcf = SelectVariants.output_vcf,
+                custom_annotations_header = MakeSubpopulationFilesAndReadSchemaFiles.custom_annotations_template_file,
+                output_vcf_name = "${vcf_filename}.unannotated.sites_only.vcf",
+                output_custom_annotations_filename = "${vcf_filename}.custom_annotations.tsv"
+        }
+
+        ## Use Nirvana to annotate the sites-only VCF and include the AC/AN/AF calculations as custom annotations
+        call AnnotateVCF {
+            input:
+                input_vcf = StripCustomAnnotationsFromSitesOnlyVCF.output_vcf,
+                output_annotated_file_name = "${vcf_filename}_annotated",
+                nirvana_data_tar = nirvana_data_directory,
+                custom_annotations_file = StripCustomAnnotationsFromSitesOnlyVCF.output_custom_annotations_file,
+        }
+
+
+        call PrepVtAnnotationJson {
+            input:
+                annotation_json = AnnotateVCF.annotation_json,
+                output_file_suffix = "${vcf_filename}.json.gz",
+                output_path = output_path,
+        }
+
+        call PrepGenesAnnotationJson {
+            input:
+                annotation_json = AnnotateVCF.annotation_json,
+                output_file_suffix = "${vcf_filename}.json.gz",
+                output_path = output_path,
+        }
```

Comment:
since we shard early in our pipeline, what's our strategy if one shard fails? do we need to start the whole process from the beginning? (this is likely not a question for this pr, just something i want us to think about)